packages/xrpl/src/utils/derive.ts

```diff
@@ -20,4 +20,5 @@ function deriveXAddress(options: {
   return classicAddressToXAddress(classicAddress, options.tag, options.test)
 }
 
-export { deriveKeypair, deriveXAddress }
+// eslint-disable-next-line import/no-unused-modules - deriveAddress is used externally
```

Comment:
This error will go away if you export it further up as well (at the `src/utils` level)